### ISSUE TITLE
The username must contain only lowercase letters, numbers, and hyphens/underscores.

### ISSUE THREAD

**Issue by ctgraham** [pkp/pkp-lib#906]
What is driving the requirement that "The username must contain only lowercase letters, numbers, and hyphens/underscores."?

With Shibboleth authentication, the best practice username would be a scoped attribute, eg. user@institution-scope (usually this is just an email address).

Are there any known dependencies that will break if we loosen the username validation?

**Comment by asmecher** [comment]
No, nothing should depend on the username having specific content. FWIW, we might also reconsider our separation of usernames and emails; I've fought against this before but I'll concede that I find that approach useful.

**Comment by asmecher** [comment]
(See essentially the same proposals at https://github.com/pkp/pkp-lib/issues/4053)

**Comment by MahreenAthar** [comment]
Hi. I'm using Ubuntu 20.04.3 LTS 64-bit and GNOME Type 3.36.8. I'm having trouble connecting Android Studio with Github. I have gone through what the naming convention is for a repository on Github but for some reason it keeps giving me this error. 
![Screenshot from 2021-11-09 12-46-53](https://user-images.githubusercontent.com/39671939/140883215-65057282-8c34-41c5-ae6e-1f3bcfac0b81.png)
Can someone help me with this. Also i have authorized my account using a personal access token. Does this have to do anything with why I'm not able to create the repo?

**Comment by MahreenAthar** [comment]
Hi. I'm using Ubuntu 20.04.3 LTS 64-bit and GNOME Type 3.36.8. I'm having trouble connecting Android Studio with Github. I have gone through what the naming convention is for a repository on Github but for some reason it keeps giving me this error. 
![Screenshot from 2021-11-09 12-46-53](https://user-images.githubusercontent.com/39671939/140883215-65057282-8c34-41c5-ae6e-1f3bcfac0b81.png)
Can someone help me with this. Also i have authorized my account using a personal access token. Does this have to do anything with why I'm not able to create the repo?

**Comment by MahreenAthar** [comment]
> Hi. I'm using Ubuntu 20.04.3 LTS 64-bit and GNOME Type 3.36.8. I'm having trouble connecting Android Studio with Github. I have gone through what the naming convention is for a repository on Github but for some reason it keeps giving me this error. ![Screenshot from 2021-11-09 12-46-53](https://user-images.githubusercontent.com/39671939/140883215-65057282-8c34-41c5-ae6e-1f3bcfac0b81.png) Can someone help me with this. Also i have authorized my account using a personal access token. Does this have to do anything with why I'm not able to create the repo?

Okay i'm able to do it now. I had to revoke the entire Personal Access Token and generate a new one and for some reason with this new one it worked.
